src/VisualStudio/Core/Def/Implementation/Venus/ContainedLanguage.IVsContainedLanguageCodeSupport.cs

```diff
@@ -210,6 +210,16 @@ protected Document GetThisDocument()
             return document;
         }
 
+        protected DocumentId FindProjectDocumentIdWithItemId(uint itemidInsertionPoint)
+        {
+            return Project.GetCurrentDocuments().SingleOrDefault(d => d.GetItemId() == itemidInsertionPoint).Id;
+        }
+
+        protected uint FindItemIdOfDocument(Document document)
+        {
+            return Project.GetDocumentOrAdditionalDocument(document.Id).GetItemId();
+        }
```

Comment:
The only caller of these is this class and derived classes, so I moved them here. I want trying to understand what breaks if you have a `ContainedLanguage` but not a `ContainedDocument`.